cluster-scope/base/core/namespaces/vault/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- namespace.yaml
```

Comment:
@larsks in OPF we give the namespace resource quotas etc here. How are those decided in this environment?